cmd/podmanV2/containers/run.go

```diff
@@ -75,6 +75,26 @@ func run(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	ie, err := registry.NewImageEngine(cmd, args)
+	if err != nil {
+		return err
+	}
+	br, err := ie.Exists(registry.GetContext(), args[0])
+	if err != nil {
+		return err
+	}
+	if !br.Value || cliVals.Pull == "always" {
+		if cliVals.Pull == "never" {
```

Comment:
Are there const for these strings?

Reply:
there are in other packages but i am concerned about the value of importing them vs bloat.  i can do it and we can see how bad it is ?